src/views/footers/local.blade.php

```diff
@@ -0,0 +1,68 @@
+<footer class="py-10 bg-gray-800">
+    <div class="px-4 text-white lg:px-16">
+        <div class="-mx-2 flex flex-wrap justify-between space-y-8 lg:space-y-0">
+            <div class="w-full px-2 lg:w-auto">
+                <a class="inline-block hover:text-gray-300 focus:outline-none focus:ring focus:ring-blue-400" href="#">
```

Comment:
I guess we need a `logo_href` prop for this link.

Reply:
Added!